mk_mvc.mak

```diff
@@ -24,7 +26,7 @@ readtags.exe: readtags.c
 
 # Debug version
 dctags.exe: respmvc
-	cl /Zi -DDEBUG /Fe$@ @respmvc debug.c /link setargv.obj
+	cl /Zi -DDEBUG /Fe$@ @respmvc /link setargv.obj
 
```

Comment:
not only for nim parser, debug.c already included in source.mak. if debug.c not removed from mk_mvc.mak, building process will fail